src/main/java/domain/card/deck/CardDeckFactory.java

```diff
@@ -1,20 +1,25 @@
-package domain.card;
+package domain.card.deck;
 
-import static java.util.stream.Collectors.collectingAndThen;
 import static java.util.stream.Collectors.toList;
 
+import domain.card.Card;
+import domain.card.Denomination;
+import domain.card.Suit;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
-public class CardFactory {
-    private CardFactory() {
+public class CardDeckFactory {
+    private CardDeckFactory() {
     }
 
     public static CardDeck createCardDeck() {
-        return Arrays.stream(Suit.values())
-                .map(CardFactory::createCards)
+        List<Card> cards = Arrays.stream(Suit.values())
+                .map(CardDeckFactory::createCards)
                 .flatMap(List::stream)
-                .collect(collectingAndThen(toList(), CardDeck::new));
+                .collect(toList());
```

Comment:
```suggestion
                .collect(Collectors.toList());
```
웬만하면 클래스명을 생략안하는게 좋을것 같습니다!

Reply:
static import 제거했습니다!